scripts/rename-schema.sh

```diff
@@ -11,6 +11,14 @@ perl -pi -e s,'Generic(.*)<FacetedUnitSpec\,LayerSpec>','\1',g build/vega-lite-s
 perl -pi -e s,'GenericUnitSpec<EncodingWithFacet\,AnyMark>','FacetedCompositeUnitSpecAlias',g build/vega-lite-schema.json
 perl -pi -e s,'GenericUnitSpec<Encoding\,AnyMark>','CompositeUnitSpecAlias',g build/vega-lite-schema.json
 
+perl -pi -e s,'FieldDefWithoutScale','FieldDef',g build/vega-lite-schema.json
```

Comment:
Do you think that we might ever accidentally rename two distinct properties that then become the same? Any idea how we could test it automatically?

Reply:
If that's the case, we will have redundant keys in `schema.definitions` and the schema will no longer be a valid json.